doc/release/launcher.rst

```diff
@@ -91,122 +94,309 @@ installation-specific configuration, e.g.), you can often use the ``-v``
 flag to capture the commands that the launcher executes on your behalf
 and customize them for your needs.
 
-
---------------------------------
 Forwarding Environment Variables
---------------------------------
+++++++++++++++++++++++++++++++++
 
 Chapel launchers generally arrange for environment variables to be
 forwarded to worker processes. However, this strategy is not always
 reliable. The remote system may override some environment variables, and
 some launchers might not correctly forward all environment variables.
 
 
----------------------------------
-_real binary suffix for execution
----------------------------------
+.. _using-udp:
 
-In order to support profiling tools that produce new binaries for the
-launcher to execute, the suffix of the real binary executed by the
-launcher may be changed with the ``CHPL_LAUNCHER_SUFFIX`` environment
-variable. If this variable is unset, the suffix defaults to "_real",
-matching the compiler's output.
+Using the Portable UDP Conduit
```

Comment:
@bradcray  - +1 to putting UDP info into its own file. 

After some discussion offline, I propose a directory/documentation hierarchy as follows:

* Platforms
   * Cray
   * OS X
   * ...
   * Network Protocols
      * Infiniband
      * UDP

I don't think network protocols is worthy of a top-level documentation directory, aka top-level sidebar item in the [online docs](http://chapel.cray.com/docs/master/index.html)